### ISSUE TITLE
Don't over-ride global whitespace cleanup options

### ISSUE THREAD

**Issue by akappen** [slim-template/language-slim#17]
I set the whitespace package to remove trailing whitespace whenever I save a buffer. This doesn't work when editing slim files because the language-slim package over-rides the global whitespace settings.

https://github.com/slim-template/language-slim/blob/master/settings/language-slim.cson

```
'.text.slim':
  'editor':
    'commentStart': '/ '
    'increaseIndentPattern': '$.'
  whitespace:
    removeTrailingWhitespace: false
    ignoreWhitespaceOnCurrentLine: false
```

Because of this I have to over-ride the over-ride in my personal config.cson file:

```
'.text.slim':
  whitespace:
    removeTrailingWhitespace: true
```

It would be less confusing if language-slim used the global whitespace settings. If there is a compelling reason for this package to over-ride the global setting, please let me know and I'll close the bug.

**Comment by pokonski** [comment]
Yes, extremely annoying. This package should not control that

**Comment by krzyzak** [comment]
:+1:

**Comment by markokajzer** [comment]
This can be closed as per [#21](https://github.com/slim-template/language-slim/pull/21).
